tools/sof_perf_analyzer.py

```diff
@@ -295,6 +296,10 @@ def parse_args():
                         help='Kernel message file captured with journalctl or other log utility')
     parser.add_argument('--out2csv', type=pathlib.Path, required=False,
                     help='Output SOF performance statistics to csv file')
+    parser.add_argument('-s', '--skip-to-first-trace', action="store_true",  default=False,
+                        help='''In CI test, some traces from previous test case will apprear in
+the mtrace of current test case, this flag is used to denote if we
+want to skip those traces, for CI test, set the flag to True''')
```

Comment:
Please don't "hide" the actual logic behind abstract names. This is test code, we need as few abstractions as possible.
```suggestion
want to skip until the first line with a timestamp between 0 and 1s. For CI test, set the flag to True''')
```